client/post-editor/controller.js

```diff
@@ -177,6 +175,15 @@ module.exports = {
 		renderEditor( context, postType );
 	},
 
+	exitPost: function( context, next ) {
+		const postId = getPostID( context );
+		const siteId = getSelectedSiteId( context.store.getState() );
```

Comment:
I'm seeing a few errors here on the Calypso errors dashboard:

>Cannot read property 'getState' of undefined

It's not clear to me when or why `context.store` would be ever be `undefined`.